### DISCUSSION TITLE
Backup Email Template

### DISCUSSION THREAD

**Discussion by mohssineAboutaj** [spatie/laravel-backup#1473] [General]
Hi everyone

Is there any way to **override** or **update** the **email** template for backup?
How can i send the backup file(s) as **attachment** 

Thanks in advance

**Comment by erikn69** [comment]
[creating-your-own-custom-notification](https://spatie.be/docs/laravel-backup/v8/sending-notifications/adding-extra-notification-channels#2-creating-your-own-custom-notification)
>"6 hours of debugging can save you 5 minutes of reading documentation"
